akka-typed/src/main/scala/akka/typed/MessageAndSignals.scala

```diff
@@ -43,14 +43,6 @@ final case object PreStart extends Signal
 final case object PreRestart extends Signal
 
 /**
- * Lifecycle signal that is fired upon restart of the Actor after replacing
- * the behavior with the fresh one (i.e. this signal is received within the
- * fresh replacement behavior).
- */
-@SerialVersionUID(1L)
-final case object PostRestart extends Signal
```

Comment:
Retaining this would mean that all behaviors that respond to PreStart also need to handle PostRestart. It can be done, but is it needed?